packages/block-editor/src/components/link-control/test/index.js

```diff
@@ -309,6 +309,103 @@ describe( 'Manual link entry', () => {
 	} );
 } );
 
+describe( 'Default search suggestions', () => {
+	it( 'should display a list of initial search suggestions when there is no search value or suggestions', async () => {
+		const searchSuggestionsSpy = jest.fn( fetchFauxEntitySuggestions );
+		const expectedResultsLength = 3; // set within `LinkControl`
+
+		act( () => {
+			render(
+				<LinkControl
+					fetchSearchSuggestions={ searchSuggestionsSpy }
+					initialSuggestions={ true }
```

Comment:
if it's a boolean should it be named as `showInitialSuggestions` or something like that? It looked like an object or something as named.

Reply:
Ok no problem. 

Assuming I rename this and rebase would you be ok to approve today?

Reply:
It's already approved right :P ?